### DISCUSSION TITLE
能否添加新的支付模式用于续费，例如银行卡转账，支付宝，微信，虚拟货币USDT都可以，能否添加一下谢谢。

### DISCUSSION THREAD

**Discussion by Origin66572** [getlantern/lantern-forum-cn#480] [Q&A]
因为本人还有六个月就要到期，想续费但是没有信用卡，能否添加上述标题里的任何一种支付模式方便我用于续费。谢谢。

**Comment by MaxMKMax** [comment]
@Origin66572 您好。您可以用这个链接支付。这个链接可以用支付宝! 谢谢您的支持。非常开心有您这么尊贵的用户。

2年: https://buy.stripe.com/eVacPZe3A5cp1oc4gO

1年: https://buy.stripe.com/7sI8zJ4t00W97MA9Bz

关于其他的支付选项我会帮您反馈给团队. 谢谢您的建议, 我们会让支付方式变得更方便的!

**Reply by Origin66572** [reply]
非常感谢，已支付完毕。

**Reply by MaxMKMax** [reply]
@Origin66572 好的. 很开心能帮助到您! 非常感谢您支持蓝灯. 有什么需要帮忙的请随时来GitHub找我. 谢谢您!
